zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties

```diff
@@ -931,17 +931,17 @@ brk.dialogue.confirmDropMessage.button.confirm.label = Drop
 brk.dialogue.confirmDropMessage.button.cancel.label = Cancel
 brk.name = Breakpoint Extension
 brk.optionspanel.name             = Breakpoints
-brk.optionspanel.option.alwaysOnTop.label	= ZAP always on top when breakpoint hit
+brk.optionspanel.option.alwaysOnTop.label	= ZAP Always on Top When Breakpoint Hit
 brk.optionspanel.option.breakmode.label	= Break Buttons Mode:
-brk.optionspanel.option.breakmode.dual.label	= Separate Request and Response buttons
-brk.optionspanel.option.breakmode.simple.label	= Single combined button
-brk.optionspanel.option.confirmDropMessage.label = Confirm drop trapped message
-brk.optionspanel.option.inScopeOnly.label = Only break on messages in scope
-brk.optionspanel.option.showBreakFilteringButtons.label = Show buttons to select the requests you don't want ZAP to break on
-brk.optionspanel.option.javaScriptUrlRegex.label = Javascript URL regex:
-brk.optionspanel.option.cssAndFontsUrlRegex.label = CSS and Fonts URL regex:
-brk.optionspanel.option.multimediaUrlRegex.label = Multimedia URL regex:
-brk.optionspanel.option.notpossibletoshowtip = Can not show this buttons when break buttons are only in the toolbar
+brk.optionspanel.option.breakmode.dual.label	= Separate Request and Response Buttons
+brk.optionspanel.option.breakmode.simple.label	= Single Combined Button
+brk.optionspanel.option.confirmDropMessage.label = Confirm Drop Trapped Message
```

Comment:
I'd suggest leaving this one and adding a period.

Reply:
Thanks. Updated commit and implemented suggestion.